lib/terraforming/resource/iam_policy_attachment.rb

```diff
@@ -16,24 +16,24 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_policy_attachment")
+        apply_template(@client, 'tf/iam_policy_attachment')
       end
 
       def tfstate
         iam_policy_attachments.inject({}) do |resources, policy_attachment|
           attributes = {
-            "id" => policy_attachment[:name],
-            "name" => policy_attachment[:name],
-            "policy_arn" => policy_attachment[:arn],
-            "groups.#" => policy_attachment[:entities].policy_groups.length.to_s,
-            "users.#" => policy_attachment[:entities].policy_users.length.to_s,
-            "roles.#" => policy_attachment[:entities].policy_roles.length.to_s,
+            'id' => policy_attachment[:name],
+            'name' => policy_attachment[:name],
+            'policy_arn' => policy_attachment[:arn],
+            'groups.#' => policy_attachment[:entities].policy_groups.length.to_s,
```

Comment:
__[rubocop]__
* Line is too long. [81/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)